packages/gatsby-theme/src/components/TutorialSidebar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Card, Image, Heading, Flex } from './shared/base';
+import { authors } from '../utils/sampleData';
+import AuthorList from './AuthorList';
+import CustomButton from './CustomButton';
+
+type SidebarProps = {
+  tutorialTitle: string;
+  chapters: string[];
+};
+
+export const Sidebar: React.FunctionComponent<SidebarProps> = ({
+  tutorialTitle,
+  chapters,
+}) => {
+  return (
+    <div>
+      <Card p={3} m={2}>
+        <Flex flexDirection="column">
+          <Heading> {tutorialTitle} </Heading>
+        </Flex>
+      </Card>
+      <ul>
+        {chapters.map(chapter => (
+          <li key={chapter}>{chapter}</li>
+        ))}
+      </ul>
+      <AuthorList authors={authors} />
+      <CustomButton type="spectrum"> Get help on Spectrum </CustomButton>
+    </div>
+  );
+};
+
+export const TabletSidebar: React.FunctionComponent<SidebarProps> = ({
+  tutorialTitle,
+  chapters,
+}) => {
+  console.log(tutorialTitle);
```

Comment:
@Allicolyer Make sure you remove console.logs

Reply:
Those are there right now as place holders. Since we don't have the designs for a phone or tablet sized screen yet I just show the hamburger icon and console.log the tutorialTitle and the chapters. Is that okay to leave in there for now?

Reply:
Yes!